### PR TITLE
New docs page

### PR DESCRIPTION
**Pull request by jonathanlei**
implement new docs page design according to https://www.figma.com/file/gxbKHEJUhGPZF2gWL8Y1Zh/Docs-and-Dev-Tools-2022?node-id=1370%3A74540

### CODE REVIEW

File: _components/pages/Docs.tsx [hunk 1]
```diff
@@ -0,0 +1,424 @@
...
+                <h5>
+                  <a
+                    className="btn-arrow-purple"
+                    target="_blank"
+                    href="https://github.com/XRPLF/xrpl.js"
+                  >
+                    Javascript
+                  </a>
+                </h5>
+              </div>
```
Comment (mDuo13) [inline _components/pages/Docs.tsx]: I think the whole block—especially the icon—should be clickable, not just the text. Here's one way you could do that (I think it'll need CSS fixes to make the hover color look right for the text though):

```suggestion
              <a className="col langs" target="_blank"
                    href="https://github.com/XRPLF/xrpl.js">
                <img
                  src={prefix + "/img/docs/javascript.svg"}
                  className="circled-logo"
                ></img>
                <h5 className="btn-arrow-purple">
                    Javascript
                </h5>
              </a>
```

File: _components/pages/Docs.tsx [hunk 2]
```diff
@@ -0,0 +1,424 @@
...
+  return (
+    <div className="col">
+      <h6>
+        {props.title}
+        <a className="btn-arrow-purple" target="_blank" href={props.link}></a>
+      </h6>
+      <p> {props.description}</p>
+    </div>
+  );
+}
```
Comment (mDuo13) [inline _components/pages/Docs.tsx]: Right now, only the little tiny arrow after the text is a hyperlink. That seems wrong. I think the whole card should be a link. For example:

```suggestion
function DevToolsCard({ props }) {
  return (
    <a className="col" target="_blank" href={props.link}>
      <h6 className="btn-arrow-purple">
        {props.title}
      </h6>
      <p>{props.description}</p>
    </a>
  );
}
```

**Review by mDuo13 — CHANGES_REQUESTED** [review]
It mostly looks good, but there are a couple places where the amount of an object that is actually a clickable link seems smaller than it should be. I like to be generous with the amount of area that is actually clickable, so users don't need to have precise dexterity to click what they wanted.

**Review by mDuo13 — APPROVED** [review]
Looks good to me now

**Review by amarantha-k — APPROVED** [review]
Looks great! Thanks @jonathanlei!

**Pull request merged**